iphone/Classes/KrollBridge.m

```diff
@@ -770,7 +770,7 @@ -(id)loadCommonJSModule:(NSString*)code withSourceURL:(NSURL *)sourceURL
 	 *      "/Resources/views/login/window.js"  __dirname = "views/login"
 	 */
 
-	NSString *filename = [sourceURL lastPathComponent];
+	NSString *filename = [sourceURL lastPathComponent]; // FIXME This should be the absolute path to the file!
```

Comment:
@sgtcoolguy isn't the absolute file path the following line? 
```
NSString *js = [[NSString alloc] initWithFormat:TitaniumModuleRequireFormat, dirname, filename,code];
```
What do you need changed here?